### ISSUE TITLE
How to finetune?

### ISSUE THREAD

**Issue by GinIsTheAnswer** [microsoft/StyleSwin#28]
Sorry if this is a silly question, but I wanted ask if you could provide and example of how to fine-tune one of your existing models to a new dataset?

An example from StyleGan3
`# Fine-tune StyleGAN3-R for MetFaces-U using 1 GPU, starting from the pre-trained FFHQ-U pickle.`
`python train.py --outdir=~/training-runs --cfg=stylegan3-r --data=~/datasets/metfacesu-1024x1024.zip \
    --gpus=8 --batch=32 --gamma=6.6 --mirror=1 --kimg=5000 --snap=5 \
    --resume=https://api.ngc.nvidia.com/v2/models/nvidia/research/stylegan3/versions/1/files/stylegan3-r-ffhqu-1024x1024.pkl
`

**Comment by TheGullahanMaster** [comment]
You supply the path to the checkpoint via `--ckpt`, ie. `--ckpt samples/000001.pt`. Also don't forget to supply original hyperparams, like image size, style dim, if we use SN etc...

**Comment by ForeverFancy** [comment]
> You supply the path to the checkpoint via `--ckpt`, ie. `--ckpt samples/000001.pt`. Also don't forget to supply original hyperparams, like image size, style dim, if we use SN etc...

Thanks, that's right.
